pkg/agent/proxy/proxier.go

```diff
@@ -453,10 +539,10 @@ func NewDualStackProxier(
 	hostname string, informerFactory informers.SharedInformerFactory, ofClient openflow.Client) k8sproxy.Provider {
 
 	// Create an ipv4 instance of the single-stack proxier
-	ipv4Proxier := NewProxier(hostname, informerFactory, ofClient, false)
+	ipv4Proxier := NewProxier(hostname, informerFactory, ofClient, true, false)
 
 	// Create an ipv6 instance of the single-stack proxier
-	ipv6Proxier := NewProxier(hostname, informerFactory, ofClient, true)
+	ipv6Proxier := NewProxier(hostname, informerFactory, ofClient, true, true)
```

Comment:
maybe a noob question, but why don't we check for the `EndpointSlice` feature gate for the dual-stack case?

Reply:
Checking feature gate is better.